docs/api/index.md

```diff
@@ -311,6 +311,7 @@ You can inject parameters with [printf formatting](https://nodejs.org/api/util.h
 - `%j`: json
 - `%o`: object
 - `%#`: index of the test case
+- `%$`: number of the test case
```

Comment:
I didn't know if people would normally call these "index" vs "number".
Would it be clearer to maybe say 0-based or 1-based index?